docs/DEVGUIDE.md

```diff
@@ -45,7 +45,7 @@ To clone the repository:
 First `cd` to the root of the cloned repository tree.
 To build the service-catalog you have two options:
 * `make build`
-* `DOCKER=1 make build`
+* `build/run.sh make build`
```

Comment:
+1 mainly because 1) it reduces the travis test time, and 2) it gets our default dev UX down to "docker + git clone ... + make all".